exercises/pascals-triangle/test/test_pascals_triangle.c

```diff
@@ -0,0 +1,88 @@
+#include <stdbool.h>
+#include "../src/pascals_triangle.h"
+#include "vendor/unity.h"
+
+bool check(int count, size_t result[][count], size_t expected[][count])
+{
+   for (int i = 0; i < count; i++) {
+      for (int j = 0; j < i; j++) {
+         if (result[i][j] != expected[i][j]) {
+            return 0;
+         }
+      }
+   }
+   return 1;
+}
+
+void test_no_rows(void)
+{
+   size_t result[1][1];
+   size_t expected[1][1];
```

Comment:
Arrays initialized here are not zeroed out, so `expected` has garbage values in it unless you explicitly save a 0 to it.